Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/release/NewReleaseInfoFragment.kt

```diff
@@ -0,0 +1,64 @@
+package de.rki.coronawarnapp.ui.release
+
+import android.os.Bundle
+import android.view.View
+import android.view.accessibility.AccessibilityEvent
+import androidx.core.view.isGone
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.navArgs
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.NewReleaseInfoScreenFragmentBinding
+import de.rki.coronawarnapp.util.di.AutoInject
+import de.rki.coronawarnapp.util.ui.doNavigate
+import de.rki.coronawarnapp.util.ui.observe2
+import de.rki.coronawarnapp.util.ui.viewBindingLazy
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactoryProvider
+import de.rki.coronawarnapp.util.viewmodel.cwaViewModels
+import javax.inject.Inject
+
+class NewReleaseInfoFragment : Fragment(R.layout.new_release_info_screen_fragment), AutoInject {
+
+    @Inject lateinit var viewModelFactory: CWAViewModelFactoryProvider.Factory
+
+    private val vm: NewReleaseInfoFragmentViewModel by cwaViewModels { viewModelFactory }
+    private val binding: NewReleaseInfoScreenFragmentBinding by viewBindingLazy()
+    private val args: NewReleaseInfoFragmentArgs by navArgs()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding.apply {
+            newReleaseInfoNextButton.setOnClickListener {
+                vm.userHasReadReleaseNotes()
```

Comment:
Is there any case where we need use `onNextButtonClick` without `userHasReadReleaseNotes` so it requires having this as two separate function calls

Reply:
You're right, I don't think so. I'll put them under the same function.